taichi/transforms/ir_printer.cpp

```diff
@@ -182,16 +182,15 @@ class IRPrinter : public IRVisitor {
   }
 
   void visit(FrontendPrintStmt *print_stmt) override {
-    print("print \"{}\", {}", print_stmt->str, print_stmt->expr.serialize());
+    print("print {}", print_stmt->contents[0].serialize());
```

Comment:
Since this is powerful enough to support multiple args, WDYT if we print the number of args as well?

Reply:
Implemented using `fmt::join`.